confgenerator/config.go

```diff
@@ -130,6 +131,16 @@ type MetricsReceiver struct {
 	CollectionInterval string `yaml:"collection_interval"` // time.Duration format
 }
 
+type MetricsProcessorExcludeMetrics struct {
+	MetricPrefixes []string `yaml:"metric_prefixes"`
```

Comment:
If these have to be globs, are they prefixes anymore?

Reply:
The question will be for @quentinmit. I have this same question too. As Ling posted the final final syntax decision in the bug, it’s metric_prefixes, but this part of code is really doing partial globs. Partial meaning the prefix has to be agent.google…../ and the suffix has to be /*. Then things in between is glob.. is that really a glob?

related comment: https://github.com/GoogleCloudPlatform/ops-agent/pull/115#discussion_r659949800

Reply:
I thought we had settled on a different name, yeah. Not sure what exactly it was... `exclude_metrics`? @qingling128 ?

Reply:
`metrics_pattern` is the one we agreed on offline. The change has been made to this PR already.